File: gungnir/plugins/src/test/resources/echo.js
var List = Java.type('java.util.List');
var Map = Java.type('java.util.Map');
var Struct = Java.type('org.gennai.gungnir.tuple.Struct');

function evaluate(value) {
  if (value instanceof List) {
    value.add("js");
    return value;
  } else if (value instanceof Map) {
    value.put("js", "value");
    return value;
  } else if (value instanceof Struct) {
    return value.getValues();
  } else {
    return "js 1:" + value;
  }
}

File: gungnir/plugins/src/test/resources/echo.py
from java.util import List
from java.util import Map
from org.gennai.gungnir.tuple import Struct

def evaluate(value):
  if isinstance(value, List):
    value.add("py");
    return value;
  elif isinstance(value, Map):
    value.put("py", "value")
    return value;
  elif isinstance(value, Struct):
    return value.getValues();
  else:
    return "py 1:" + str(value)